File: Practice/CPP/2023-9-25-sort/SortUtils.cpp
#include"Head.h"

void BubbleSort(int *array, int size){
	for(int i = 0; i < size; i++){
		for(int j = 0; j < size - i - 1; j++){
			if(array[j] > array[j+1]) Swap(array[j],array[j+1]);
		}
	}		
}

File: Code/CS/2023-10-1-dll/MyLib/SortUtils.cs
using System;
using Utils;

namespace SortUtils{

	public class SortUtils{

		public void BubbleSort(int[] arr){
	
			Utils.Utils utils = new Utils.Utils();	
			for(int i = 0; i < arr.Length; i++){
				for(int j = 0; j < arr.Length - i - 1; j++){
					if(arr[j] > arr[j+1]){
						utils.Swap(ref arr[j], ref arr[j+1]);
					}
				}
			}
		}
	}
}
